src/main/src/main/java/org/geoserver/catalog/ResourcePool.java

```diff
@@ -1941,9 +1940,7 @@ private HTTPClient getHTTPClient(HTTPStoreInfo info) {
                 mtClient.setMaxConnections(maxConnections);
             }
         } else {
-            client =
-                    HTTPClientFinder.createClient(
-                            new Hints(Hints.HTTP_CLIENT, SimpleHttpClient.class));
+            client = HTTPClientFinder.createClient();
```

Comment:
From a mere user point of view, the two calls seem equivalent. But I guess they are not? What is happening here?